train.py

```diff
@@ -26,7 +27,7 @@
 flags.DEFINE_string('classes', './data/coco.names', 'path to classes file')
 flags.DEFINE_enum('mode', 'fit', ['fit', 'eager_fit', 'eager_tf'],
                   'fit: model.fit, '
-                  'eager_fit: model.fit(run_eagerly=True), '
+                  'eager_fit: model.fit(RUN_EAGERLY=True), '
```

Comment:
is there a reason this param is captilized?

Reply:
Not sure why that got changed to all caps, but that should be corrected now in the latest commit.